### PR TITLE
script to set update time for all _review

### PR DESCRIPTION
**Pull request by JiaoJiao123**

### CODE REVIEW

File: web/yo/app/scripts/controllers/genes.js [hunk 1]
```diff
@@ -131,6 +131,85 @@ angular.module('oncokbApp')
...
+            };
+            function fetchGeneDoc(index) {
+                if(index < $scope.documents.length) {
+                    var fileId = $scope.documents[index].id;
+                    storage.getRealtimeDocument(fileId).then(function(realtime) {
+                        if (realtime && realtime.error) {
+                            console.log('did not get realtime document.');
+                            $timeout(function() {
+                                fetchGeneDoc(++index);
+                            },    2000, false);
```
Comment (zhx828) [inline web/yo/app/scripts/controllers/genes.js]: 2 second is a little bit too long.

File: web/yo/app/scripts/controllers/genes.js [hunk 2]
```diff
@@ -131,6 +131,85 @@ angular.module('oncokbApp')
...
+            function setUpdateTimeGene() {
+                $scope.geneRealtimeModel.beginCompoundOperation();
+                checkAndSet($scope.geneDoc.summary_review);
+                checkAndSet($scope.geneDoc.type_review);
+                checkAndSet($scope.geneDoc.background_review);
+                for (var i = 0; i < $scope.geneDoc.mutations.length; i++) {
+                    var mutation = $scope.geneDoc.mutations.get(i);
+                    checkAndSet(mutation.name_review);
+                    checkAndSet(mutation.oncogenic_review);
+                    checkAndSet(mutation.shortSummary_review);
```
Comment (zhx828) [inline web/yo/app/scripts/controllers/genes.js]: I think we don't need shorSummary anymore?

File: web/yo/app/scripts/controllers/genes.js [hunk 3]
```diff
@@ -131,6 +131,85 @@ angular.module('oncokbApp')
...
+                        checkAndSet(tumor.progImp_review);
+                        checkAndSet(tumor.cancerTypes_review);
+                        checkAndSet(tumor.nccn_review);
+                        checkAndSet(tumor.nccn.therapy_review);
+                        checkAndSet(tumor.nccn.disease_review);
+                        checkAndSet(tumor.nccn.version_review);
+                        checkAndSet(tumor.nccn.description_review);
+                        for (var k = 0; k < tumor.TI.length; k++) {
+                            var ti = tumor.TI.get(k);
+                            checkAndSet(ti.name_review);
```
Comment (zhx828) [inline web/yo/app/scripts/controllers/genes.js]: No need to set ti. ti wont be changed

File: web/yo/app/scripts/controllers/genes.js (hunk 2 above)
Comment (JiaoJiao123) [inline web/yo/app/scripts/controllers/genes.js]: yes, right. This script was written before we combine two sections. I will double check and update it.

**Pull request closed without merging**